core/src/test/java/org/apache/iceberg/MetadataTableScanTestBase.java

```diff
@@ -79,24 +80,25 @@ protected void validateTaskScanResiduals(TableScan scan, boolean ignoreResiduals
     }
   }
 
-  protected void validateIncludesPartitionScan(
-      CloseableIterable<FileScanTask> tasks, int partValue) {
-    validateIncludesPartitionScan(tasks, 0, partValue);
+  /** Used for asserting on data files where partition size = 1 */
```

Comment:
Thanks !

I think name /comment is a bit misleading.  The partition size  = 1 seems misleading (to me, it means how many partitions..), I think we want to mean that partition fields size is 1.  If we change the name, how about validateSingleFieldPartition(), and then maybe fix/remove the comment (remove if its a redundant).

Reply:
`validateSingleFieldPartition` it is and removed redundant comment